TestResultSummaryService/BuildProcessor.js

```diff
@@ -113,9 +113,9 @@ class BuildProcessor {
                         _id: task._id,
                         status: "Done"
                     });
+                    await plugins.onBuildDone(task, { testResultsDB, logger });

```

Comment:
@llxia I'm slightly confused about the plugin feature. If there are multiple implementations of `onBuildDone` under `TestResultSummaryService/plugins/`, we would run all the implementations sequentially? In future, if there are more implementations of `onBuildDone` under plugin, does it make sense to run all implementations?

For example, we already have this:
https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/1421cd6c5f994fd221a64f8b39adfba3f6c4048c/TestResultSummaryService/plugins/examplePlugin.js#L3

After we add `plugins/DataManagerPerf.js` with `onBuildDone`, we'll have 2 implementations. Looking at the output, we see both of them are being called. 

https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/master/TestResultSummaryService/plugins/README.md

Reply:
yes, different users could have their own implementation of `onBuildDone`. The core application just calls the `onBuildDone` inside the plugin folder.